tests/functional/ui/test_health_overview.py

```
@@ -482,7 +589,16 @@ def test_health_score_changes_based_on_alert_severity(
         assert (
             baseline_score == self.get_health_score_ui()
         ), "Full health score not recovered"
-
+        if (
+            alert_name == constants.ALERT_CLUSTERWARNINGSTATE
+            or alert_name == constants.ALERT_CLUSTERERRORSTATE
```

Ruff (0.15.13)

[warning] Consider merging multiple comparisons. Use a `set` if the elements are hashable.

Merge multiple comparisons

(PLR1714)